cli/src/main.rs

```diff
@@ -207,6 +208,49 @@ fn run() -> Result<(), CliError> {
                                 .possible_values(&["accept", "reject"])
                                 .help("Reject the proposal"),
                         ),
+                )
+                .subcommand(
+                    SubCommand::with_name("list")
+                        .about("List the circuits")
+                        .arg(
+                            Arg::with_name("url")
+                                .short("u")
+                                .long("url")
+                                .help("The url of the Splinter Daemon REST API")
```

Comment:
URL, daemon (per capitalization guidelines) here and below.

Reply:
updated